src/west/app/project.py

```diff
@@ -315,6 +317,43 @@ def check_call(self, args, cwd=None):
                 level=log.VERBOSE_VERY)
         subprocess.check_call(args, cwd=cwd)
 
+    def check_output(self, args, cwd=None):
+        cmd_str = util.quote_sh_list(args)
+        log.dbg(f"running '{cmd_str}' in {cwd or os.getcwd()}",
+                level=log.VERBOSE_VERY)
+        return subprocess.check_output(args, cwd=cwd)
```

Comment:
I'm surprised `west` had nothing like this already.

Reply:
> I'm surprised `west` had nothing like this already.

We would normally use Project.git(), but we can't, because we haven't bootstrapped anything yet.